### PR TITLE
Chapter 18-6 18_6_Accediendo_a_Bitcoind_con_Swift.md

### PR DESCRIPTION
**Pull request by javiervargas**

### CODE REVIEW

File: es/18_6_Accediendo_a_Bitcoind_con_Swift.md [hunk 1]
```diff
@@ -0,0 +1,458 @@
...
+
+        let unspent = result as! NSArray
+        let utxo = unspent[0] as! NSDictionary
+        
+        let txid = utxo["txid"] as! NSString
+        let vout = utxo["vout"] as! NSInteger
+        let amount = utxo["amount"] as! NSNumber
+        let new_amount = amount.floatValue - 0.0001
+```
+Como en otros ejemplos, usted va a tomar arbitrariamente el 0º UTXO, y usara las `txid`, `vout`, y `amount` de ella.
```
Comment (icculp) [inline es/18_6_Accediendo_a_Bitcoind_con_Swift.md:339]: usar o usará?

File: es/18_6_Accediendo_a_Bitcoind_con_Swift.md [hunk 2]
```diff
@@ -0,0 +1,458 @@
...
+```
+        method = "createrawtransaction"
+        param="[ { \"txid\": \"\(txid)\", \"vout\": \(vout) } ], { \"\(address)\": \(new_amount)}"
+        makeCommand(method: method,param: param) { result in
+
+            let hex = result as! NSString
+```
+### 3. Firme la transacción cruda
+
+La firma de su transacción sólo requiere que ejecute la transacción `signrawctionwithwallet`RPC, usando su nuevo `hex`:
```
Comment (icculp) [inline es/18_6_Accediendo_a_Bitcoind_con_Swift.md:355]: \`signrawctionwithwallet\` RPC,

**Review by icculp — CHANGES_REQUESTED** [review]

**Review by icculp — APPROVED** [review]

**Comment by icculp** [comment]
@shannona ready to be merged

**Comment by shannona** [comment]
Terrific.

**Pull request merged**